.github/workflows/release.yml

```diff
@@ -150,6 +150,7 @@ jobs:
 
       - name: Create release
         id: create-release
+        # TODO: Replace as it has been deprecated
```

Comment:
It seems that this should be replaced by `gh cli`.

Example solution:
```yml
      - name: Create release
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          tag: ${{ github.ref_name }}
        run: |
          gh release create "$tag" \
              --repo="$GITHUB_REPOSITORY" \
              --title="${GITHUB_REPOSITORY#*/} ${tag#v}" \
              --generate-notes
```

Reply:
GitHub also showed some [alternatives actions in the readme](https://github.com/actions/create-release?tab=readme-ov-file#github-action---releases-api)